tools/osbuild-mpp

```diff
@@ -1200,6 +1219,9 @@ class ManifestFile:
         name = desc.get("id", "image")
         self.vars[name] = Image.from_dict(desc)
 
+    def get_pipeline_name(self, node):
+        return node.get("name", "unknown")
```

Comment:
You now switch from an empty string to `"unknown"`. I don't mind it, but it does break existing users, _iff_ they relied on this, right?

Reply:
That is true, I think we should probably revert to the existing one.